extensions/cache/src/main/resources/benchmark-redis.xml

```diff
@@ -0,0 +1,69 @@
+<benchmark xmlns="urn:radargun:benchmark:3.0">
+
+   <master bindAddress="${master.address:127.0.0.1}" port="${master.port:2103}"/>
+
+   <clusters>
+      <cluster size="8">
+         <group name="server" size="6" />
+         <group name="client" size="2" />
+      </cluster>
+   </clusters>
+
+   <configurations>
+      <config name="redis">
+         <setup group="server" plugin="redis32">
+            <server xmlns="urn:radargun:plugins:redis32:3.0" file="${env.RADARGUN_DISTRO_PATH}/plugins/redis32/conf/redis.conf"
```

Comment:
Documented?
The contents of `conf` dir are added to the classpath; RadarGun should check if it's there and resolve that for the embedded lib. Other plugins don't need absolute paths either.
`ISPN_HOME` et all are fine; there's no other place to set it. But the distro itself can detect where it is automatically, so the least you could do is setting such env var in those scripts.